core/src/main/scala/kafka/javaapi/consumer/ConsumerConnector.java

```diff
@@ -17,72 +17,72 @@
 
 package kafka.javaapi.consumer;
 
-
-import java.util.List;
-import java.util.Map;
-
 import kafka.common.OffsetAndMetadata;
 import kafka.common.TopicAndPartition;
 import kafka.consumer.KafkaStream;
 import kafka.consumer.TopicFilter;
 import kafka.serializer.Decoder;
 
+import java.util.List;
+import java.util.Map;
+
 public interface ConsumerConnector {
-  /**
-   *  Create a list of MessageStreams of type T for each topic.
-   *
-   *  @param topicCountMap  a map of (topic, #streams) pair
-   *  @param keyDecoder a decoder that decodes the message key
-   *  @param valueDecoder a decoder that decodes the message itself
-   *  @return a map of (topic, list of  KafkaStream) pairs.
-   *          The number of items in the list is #streams. Each stream supports
-   *          an iterator over message/metadata pairs.
-   */
-  public <K,V> Map<String, List<KafkaStream<K,V>>> 
-    createMessageStreams(Map<String, Integer> topicCountMap, Decoder<K> keyDecoder, Decoder<V> valueDecoder);
-  
-  public Map<String, List<KafkaStream<byte[], byte[]>>> createMessageStreams(Map<String, Integer> topicCountMap);
+    /**
```

Comment:
@granthenke Heh, I didn't even mean for the actual credit. But sometimes it's useful in tracking down where an issue was first introduced, the reasoning for the way a particular chunk of code is written, etc. It's an inconvenience, not a serious problem, especially for a set of changes this size.